src/Cover/DefaultPdfGenerator.php

```diff
@@ -443,6 +443,22 @@ protected function generalMetadataFile($document, $tempFilename = '')
 
         $documentMetadata = [];
 
+        $config = Config::get();
+
+        if (isset($config->name)) {
```

Comment:
Das reicht erst einmal. Wenn weitere Variablen aus der Konfiguration dazu kommen, sollte eine generische Lösung gefunden werden, bei der nur noch eine Liste mit Config-Optionen, die verfügbar sein sollen, definiert werden muss. Das ist machbar. 

Die Template-Variablen sollten daher auch eher "config-NAME_OF_OPTION" heißen, um die Beziehung zur Konfiguration deutlich zu machen, also "config-url" und "config-name". Der Dot in Optionsnamen, könnte je nach Bedarf für die Templates in Unterstrich oder Bindestrich umgewandelt werden.

Reply:
"config-" Präfix für Konfigurationsvariablen: 7b27aad